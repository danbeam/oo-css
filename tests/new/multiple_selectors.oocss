div.class {
    color: blue;
    #id, .class, div > span, input[type=submit], *:first-child+html {
        background: url(some/image/that/doesnt/exist.png);
    }
}
