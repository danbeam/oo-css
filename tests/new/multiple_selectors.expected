div.class {
    color: blue;
}
div.class #id, div.class .class, div.class div > span, div.class input[type=submit], div.class *:first-child+html {
    background: url(some/image/that/doesnt/exist.png);
}
